cobra2snooty_test.go

```diff
@@ -130,6 +133,10 @@ func TestGenDocs(t *testing.T) {
 
 	checkStringContains(t, output, Echo().Long)
 	checkStringContains(t, output, Echo().Example)
+	checkStringContains(t, output, fmt.Sprintf(`.. code-block::
+	\n   %s\n`, Echo().Example[IntroText]))
```

Comment:
for now feel free to hardcode, test is ok to use hardcoded values because they will catch errors easier 
```suggestion
	\n   # Example with intro text\n`))
```